### ISSUE TITLE
Suggestion : Add an option to remove the links, then directly modify the file.

### ISSUE THREAD

**Issue by WinnerWind** [adiron/obsidian-strip-internal-links#1]
Considering this plugin's use case, it is useless when used in conjunction with macros (such as Commander). By sending the result to the clipboard, this creates an extra step for users to then paste that content into a new file, creating some hassle.
It would be nice to include a option that instead modifies the file directly rather than sending the result to the clipboard. This will also make the plugin compatible with Commander (Which can't paste clipboard content.)

**Comment by adiron** [comment]
Hello @WinnerWind.

Thank you for opening my first issue in this project :) I didn't actually know about Commander either, so thank you for that too!

What you are suggesting does not sound particularly difficult. I may get to it at some point, if the stars line up. Failing that however, if you have any code skills I urge you to try your luck at implementing this yourself and I'll gladly go over your pull request.

Have a great weekend :)

**Comment by WinnerWind** [comment]
Hi there!
I'm not exactly a professional programmer (I've coded very basic Unity games, that's it). I'll still give it a try if the time allows it.
The main reason I suggested modifying the file and also giving it a command pallette command is for Commander support. Using Commander, one can make a macro like this

Copy active file -> Remove links (using your plugin) -> Export to PDF -> Delay ->Delete file
That way, no extra file is created, and the actual file itself is preserved. 

Of course, it's all in the implementation! I'll try to see if I can bodge together something, if my schedule allows it.
